### DISCUSSION TITLE
Any milestone to integrate this into IDEA/Editor-like plugins?

### DISCUSSION THREAD

**Discussion by mochadwi** [ejoffe/spr#251] [Ideas]
Simplifying process of (re)opening browser, and our editor seems a lot todo (context-switching), thanks for building this, is there any milestone to have a minimal support in IDEA/Editor-like plugins?

**Comment by ejoffe** [comment]
No current milestone like that, contributions always welcome if you feel inspired. 
There is an extension for vscode written by Adam: https://marketplace.visualstudio.com/items?itemName=adambh.vscode-git-spr

**Comment by leoluk** [comment]
You can add spr commands as External Tool and add a hotkey.
